docs/guide/usage.rst

```diff
@@ -23,7 +23,7 @@ Let's learn first how to use Entity Embed for Deduplication. After that, we can
 Preparing the data
 ~~~~~~~~~~~~~~~~~~
 
-Your data needs to represent a list of records as list of ``dict`` s. Those ``dict`` s must contain:
+Your data needs to represent a list of records as list of ``dicts``. Those ``dicts`` must contain:
```

Comment:
Thanks, this was in purpose but it's strange, better to fix it.
Could you please search for other occurrences of "`` s" and fix them? There's an "``id`` s" case.